src/components/sessions/activeSession/feedback/FeedbackChannel.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Checkbox, Header } from 'semantic-ui-react'
+
+import Feedback from './Feedback'
+
+import withCSS from '../../../../lib/withCSS'
+
+const FeedbackChannel = ({ data, intl }) => (
+  <div>
+    <Header
+      dividing
+      as="h2"
+      content={intl.formatMessage({
+        defaultMessage: 'Feedback-Channel',
+        id: 'pages.runningSession.feedbackChannel.title', // TODO correct naming of identifier
+      })}
+    />
+    <Checkbox
+      toggle
+      label={intl.formatMessage({
```

Comment:
I think we can always pass in components into labels and content props of semantic. So we should be able to use `<FormattedMessage>`.